### ISSUE TITLE
本主题的名称到底是 hexo-theme-indigo 还是 hexo-theme-material-indigo ？

### ISSUE THREAD

**Issue by ChungZH** [yscoder/hexo-theme-indigo#460]
rt，有点混乱。

**Comment by yulimchen** [comment]
> rt，有点混乱。

下载下来文件夹命名 indigo 即可。

**Comment by ChungZH** [comment]
不是 :joy: 我们现在在做 [EasyHexo](https://easyhexo.com)，写到 [indigo](https://easyhexo.com/2-Theme-use-and-config/2-10-hexo-theme-indigo/) 的时候名称有点混乱，就想统一一下～

**Comment by ChungZH** [comment]
@yscoder 麻烦过来看看～谢谢！

**Comment by yulimchen** [comment]
> 不是 😂 我们现在在做 [EasyHexo](https://easyhexo.com)，写到 [indigo](https://easyhexo.com/2-Theme-use-and-config/2-10-hexo-theme-indigo/) 的时候名称有点混乱，就想统一一下～

老弟你真的只有 11 岁？？？！ 😂

**Comment by ChungZH** [comment]
> 
> 
> > 不是 😂 我们现在在做 [EasyHexo](https://easyhexo.com)，写到 [indigo](https://easyhexo.com/2-Theme-use-and-config/2-10-hexo-theme-indigo/) 的时候名称有点混乱，就想统一一下～
> 
> 老弟你真的只有 11 岁？？？！ 😂

是的 :smiley:

**Comment by yulimchen** [comment]
> > > 不是 😂 我们现在在做 [EasyHexo](https://easyhexo.com)，写到 [indigo](https://easyhexo.com/2-Theme-use-and-config/2-10-hexo-theme-indigo/) 的时候名称有点混乱，就想统一一下～
> > 
> > 
> > 老弟你真的只有 11 岁？？？！ 😂
> 
> 是的 😃

我可以加一下你的 QQ 吗？有机会可以请教一下你 Hexo 的问题😂

**Comment by ChungZH** [comment]
@yulimchen `QQ: 124694738`
